filament/backend/include/backend/DriverEnums.h

```diff
@@ -960,29 +960,6 @@ struct PolygonOffset {
 struct StencilState {
     using StencilFunction = SamplerCompareFunc;
 
-    StencilState() noexcept : referenceValue(0u), stencilWrite(false), padding(0u) {
-        static_assert(sizeof(StencilOperations) == sizeof(uint32_t), "StencilOperations size not what was intended");
-        static_assert(sizeof(StencilState) == 12u, "StencilState size not what was intended");
```

Comment:
As long as we have padding bytes, it's probably a good idea to keep this assertions.

Reply:
Oh just kidding, I see you put them below 😄

Reply:
I just moved it.